### PR TITLE
Remove OSG debug packages

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

**Comment by MehdiChinoune** [comment]
any objection?

File: mingw-w64-osgbullet-git/PKGBUILD [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
-pkgrel=2
+pkgrel=3
 pkgdesc="Bullet physics and OpenSceneGraph integration (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 license=('spdx:LGPL-2.1')
 url="https://github.com/mccdo/osgbullet/"
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}" "${MINGW_PACKAGE_PREFIX}-${_realname}-svn")
+replaces=(${MINGW_PACKAGE_PREFIX}-${_realname}-svn)
```
Comment (lazka) [inline mingw-w64-osgbullet-git/PKGBUILD:16]:
```suggestion
provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

File: mingw-w64-osgqtquick-git/PKGBUILD [hunk 2]
```diff
@@ -2,24 +2,26 @@
...
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clangarm64' 'clang32')
-license=('LGPL')
+license=('spdx:MIT')
 url="https://github.com/podsvirov/osgqtquick"
-makedepends=("git" 
-             "make"
-             "${MINGW_PACKAGE_PREFIX}-cc"
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}" "${MINGW_PACKAGE_PREFIX}-${_realname}-svn")
+replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}-svn")
```
Comment (lazka) [inline mingw-w64-osgqtquick-git/PKGBUILD:17]:
```suggestion
conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

File: mingw-w64-osgworks-git/PKGBUILD [hunk 3]
```diff
@@ -2,22 +2,24 @@
...
-pkgrel=5
+pkgrel=6
 pkgdesc="A toolkit for OpenSceneGraph applications (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
 license=('spdx:LGPL-2.1')
 url="https://github.com/mccdo/osgworks/"
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}" "${MINGW_PACKAGE_PREFIX}-${_realname}-svn")
+replaces=("${MINGW_PACKAGE_PREFIX}-${_realname}-svn")
```
Comment (lazka) [inline mingw-w64-osgworks-git/PKGBUILD:16]:
```suggestion
conflicts=("${MINGW_PACKAGE_PREFIX}-${_realname}")
```

**Pull request merged**